com.scanoss.sbom-workbench.yml

```diff
@@ -0,0 +1,44 @@
+app-id: com.scanoss.sbom-workbench
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: com.scanoss.sbom-workbench
+rename-desktop-file: scanoss-workbench.desktop
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=pulseaudio"
+  - "--share=network"  
+  - "--device=dri"    
+  - "--persist=.scanoss"
+  - "--persist=scanoss-workspace"
+  - "--persist=.pki"
+  - "--filesystem=/tmp:rw"
+modules:
+  - name: sbom-workbench
+    buildsystem: simple
+    build-commands:
+      - chmod +x sbom-workbench-1.12.4-linux-x86_64.AppImage
+      - ./sbom-workbench-1.12.4-linux-x86_64.AppImage --appimage-extract
+
+      - install -dm755 "${FLATPAK_DEST}/bin" "${FLATPAK_DEST}/lib/sbom-workbench"
+      - cp -r squashfs-root/* "${FLATPAK_DEST}/lib/sbom-workbench"      
+
+      - install -Dm644 "com.scanoss.sbom-workbench.png" "${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/com.scanoss.sbom-workbench.png"
+      - install -Dm755 "run.sh" "${FLATPAK_DEST}/bin/com.scanoss.sbom-workbench"
+      - desktop-file-install --dir="${FLATPAK_DEST}/share/applications" --remove-key=X-AppImage-Version --set-key=Exec --set-value=com.scanoss.sbom-workbench --set-key=Icon --set-value=com.scanoss.sbom-workbench squashfs-root/scanoss-workbench.desktop
+
+      - install -Dm644 -t ${FLATPAK_DEST}/share/metainfo com.scanoss.sbom-workbench.metainfo.xml
+    sources:
+      - type: file
+        url: https://github.com/scanoss/sbom-workbench/releases/download/v1.12.4/sbom-workbench-1.12.4-linux-x86_64.AppImage
```

Comment:
why not building from source? that would bring aarch64 support.

Reply:
Hi Hubert!  Thank you for taking a look at our request.

We've asked for instructions at flathub@lists.freedesktop.org and we received help from Bartłomiej. 

He took a look at our project and although he mentioned the possibility of building from sources in your infrastructure, he thinks it will be painful and he didn't recommend this approach for our project. He presented us with an alternative to repackage our AppImage package.

We definitely intend to add this to increase visibility and will add it our backlog

As I informed Bartłomiej, there's a Spanish KDE developers event this weekend, and we would like to showcase the SBOM Workbench available at Flathub (or nearby). It would mean a lot to us to have this opportunity.

Do you think we could plan to support aarch64 in a future release?

Also, any other suggestion is welcomed. Thank you very much for your assistance.